Makefile

```diff
@@ -60,7 +60,7 @@ KERNEL_CONFIG=tools/kernel-configs/microvm-kernel-$(host_arch)-$(KERNEL_VERSION)
 # Copied from https://github.com/firecracker-microvm/firecracker/blob/v1.0.0/tools/devtool#L2015
 # This allows us to specify a kernel without the patch version, but still get the correct build path to reference the kernel binary
 KERNEL_FULL_VERSION=$(shell cat "$(KERNEL_CONFIG)" | grep -Po "^\# Linux\/$(kernel_config_pattern) (([0-9]+.){2}[0-9]+)" | cut -d ' ' -f 3)
```

Comment:
I normally prefer to clean up unused defined variables. In this case looks like `KERNEL_FULL_VERSION` is no longer being used. I will let you decide if we need to keep this around for future use.

Reply:
found the real reason - the regex for kernel version slightly changed in firecracker v1.1.0: https://github.com/firecracker-microvm/firecracker/blob/v1.1.0/tools/devtool#L2082 . updated the regex in our Makefile and reverted back to using `KERNEL_FULL_VERSION`